packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -344,6 +345,27 @@ export interface Behavior {
    */
   readonly maxTtlSeconds?: number;
 
+  /**
+   * Declares associated lambda@edge functions for this distribution behaviour.
+   *
+   * @default undefined
+   */
+  readonly lambdaFunctionAssociations?: LambdaFunctionAssociation[];
+
+}
+
+export interface LambdaFunctionAssociation {
+
+  readonly eventType?: LambdaEdgeEventType;
+
+  readonly lambdaFunction?: lambda.IVersion;
```

Comment:
MUST this be a version? Could it also be LATEST, i.e., `lambda.IFunction` ?

Reply:
To use lambda@edge it must be a version (as far as I know). The LATEST will not be accepted by cloudfoundation.